_opensearch/rest-api/index-apis/put-mapping.md

```diff
@@ -1,21 +1,24 @@
 ---
 layout: default
-title: Put mapping
+title: Create or update mappings
 parent: Index APIs
 grand_parent: REST API reference
-nav_order: 200
+nav_order: 220
 ---
 
-# Put mapping
+# Create or update mappings
 Introduced 1.0
 {: .label .label-purple }
 
-The put mapping API operation lets you add new mappings and fields to an index.
+If you want to create or add mappings and fields to an index, you can use the put mapping API operation.
+
+You can't use this operation to update mappings that already map to existing data in the index. You must first create a new index with your desired mappings, and then use the [reindex API operation]({{site.url}}{{site.baseurl}}/opensearch/reindex-data) to map all the documents from your old index to the new index. If you don't want any downtime while you re-index your indexes, you can use [aliases]({{site.url}}{{site.baseurl}}/opensearch/index-alias).
 
```

Comment:
Can we add a sentence here saying something like, "If your mapping already exists, this operation updates the mapping instead."?

Reply:
ok thanks